conf/base.config

```diff
@@ -59,7 +59,4 @@ process {
         errorStrategy = 'retry'
         maxRetries    = 2
     }
-    withName:CUSTOM_DUMPSOFTWAREVERSIONS {
-        cache = false
-    }
 }
```

Comment:
@skrakau nf-core would remove this label completely. Whast do you think about this? I've seen that you added the cache lenient label to some processes. Should we remove those labels as well?

Reply:
phew... why was this added in the first place? 
There were some issues in the past: https://github.com/qbic-pipelines/metapep/pull/48
not sure if this does not occur anymore. Then it should be removed.

Reply:
Why did nf-core decide on removing this? do you have a link to a discussion or so?

Reply:
I'm not sure if it was a nf-core label in the first place! We could change the corresponding processes and add `cache = 'lenient'` or add it to the institute config if it is cfc2 specific? It should not only be metapep if the resume is not functioning properly due to timestamps on `scratch` right?

Reply:
If we add it to the processes there will be less merge conflicts, and. as the label only includes this one parameter I think it's not beneficial to keep it as label!

Reply:
it might also be the case that it was mainly a problem for this particular problem due to it's large amounts of run tasks.

Reply:
not sure, before adding it for each individual process, I would check if this still occurs maybe

Reply:
it also occurred on `cfc` (not on `cfc2`) at that time

Reply:
Oh true, that was on cfc! I'll test again if it occurs on cfc2 without lenient cache!

Reply:
Tested on cfc2 without issues. If it will become a problem again the it can be added globally to the base.config or using a custom config.